storage/postgres.go

```diff
@@ -1,7 +1,45 @@
-// Package storage provides implementation for persisting user information.
-// It includes the following methods:
-// - create new user
-// - get information about exists user
-// - update exists user
-// - delete unnecessary user
+// Package storage has to provide an interface to create a new postgres client
+// with ability to reconnect after connection failure.
 package storage
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	// _ mean that we can use all function from this package
```

Comment:
Do you need it here?

Reply:
Yes, I think so. This is massage from linter, without this comment: "A blank import should be only in a main or test package, or have a comment justifying it."

Reply:
for this case you can use another type of comments
//nolint:{linterName}

Reply:
I think that my comment is more informative.

Reply:
This comment says absolutely opposite to real behavior.

Reply:
My falt. Canged it.